src/components/Main/Main.js

```diff
@@ -0,0 +1,32 @@
+import { useCountries } from '../../hooks/useCountries.js';
+import Countries from '../Countries/Countries';
+import './Main.css';
+
+export default function Main() {
+  const { countryFilter, type, setType } = useCountries();
+  return (
+    <div className="country">
+      <div className="hello">
+        <select
```

Comment:
this ```<select>``` would be a great candidate to be broken out into a separate Select component